.github/workflows/release_pr.yml

```diff
@@ -0,0 +1,34 @@
+name: Create Release PR
+
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
I don't think I quite understand this "on" command. To me, it seems like the `Create Release PR` workflow would run every time a PR was closed. What am I missing here?

Reply:
Yes, so this workflow will run every time a PR is closed, but the job will only start if the prerelease tag was on the PR. there is an if on the job that checks to make sure the pr was both merged (not just closed), and also has the prerelease tag
```
jobs: 
  create_prerelease:
    if: ${{ (github.event.pull_request.merged == true) && (contains(github.event.pull_request.labels.*.name, 'prerelease')) }}
```
So it will check every PR that closes, but it won't actually run unless its the right PR.

Reply:
Got it, totally missed those lines. Thanks for clarifying!